SQL_testing.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="bg">
+	<head>
+		<meta charset="UTF-8"/>
+		<title>JAVA_01</title>
+		
+		<style>
+			body
+			{
+				border: 2px solid black;
+				text-align: center;
+			}
+		</style>
```

Comment:
Internal styles are not a good practice. You can create a separate file for css only.